locale/en/submission.po

```diff
@@ -2043,6 +2043,28 @@ msgstr "You must assign at least one participant to this submission before initi
 msgid "submission.query.allowedEditTime"
 msgstr "You can update this discussion for {$allowedEditTimeNoticeLimit} minutes."
 
+msgid "submission.list.changeSubmissionLanguage.currentLanguage"
```

Comment:
Just flagging this as another place where the workflow/author dashboard styling is happening.

Reply:
@ewhanson, what do you exactly mean?

Reply:
Sorry, @bozana. It's actually in the line below. The styling for how it appears in the UI is applied via a `<strong>` tag that's baked into the localized string.

Reply:
@ewhanson, how should this be actually solved? Having different span element with different classes and having only 'Current Submissino Language:' in the locale key? :-)
Thanks!